cli/go.mod

```diff
@@ -6,16 +6,15 @@ require (
 	cloud.google.com/go v0.38.0
 	github.com/briandowns/spinner v1.6.1
 	github.com/forseti-security/config-validator v0.0.0-20200505040130-17dc60b21dc8
+	github.com/gammazero/workerpool v1.1.2
 	github.com/golang/protobuf v1.3.3
-	github.com/google/go-cmp v0.3.1 // indirect
-	github.com/hashicorp/terraform v0.12.2 // indirect
```

Comment:
Why was this removed?

Reply:
no idea where this came from, I ran go mod tidy, maybe thats where it creeped in from?

Reply:
Ack, if everything is still building it looks fine.